plugin/src/asserts.ts

```diff
@@ -0,0 +1,5 @@
+export function assertIsDefined<T>(value: T): asserts value is NonNullable<T> {
+  if (value === undefined || value === null) {
+    throw new Error(`${value} is not defined`);
```

Comment:
When value is `undefined` or `null`, the error message will be "undefined is not defined" or "null is not defined", respectively, which is not very helpful. Maybe we should pass the error message as the second argument?
```ts
assertIsDefined(inputMap, "inputMap is not defined");
```
Also, take a look at Node.js' [`assert`](https://www.w3schools.com/nodejs/met_assert.asp), but I'm not sure how this behaves in the production build of the plugin.

Reply:
😅 I don't know how I didn't see that coming. I was thinking about adding more sophisticated assert functions later on. I think instead of passing the whole message, it would be better to just pass the name of the variable.